### DISCUSSION TITLE
What should I put in a profile for a program to ensure it doesn't have clipboard access?

### DISCUSSION THREAD

**Discussion by fury182** [netblue30/firejail#5076] [Q&A] [answered]
What should I put in a profile for a program to ensure it doesn't have clipboard access? Is this doable?

I'm on plain vanilla Ubuntu 20.04 LTS.

Thanks!

Relates to:

* #5075

**Comment by rusty-snake** [comment, marked as answer]
clipboard access comes and goes with access to the Wayland/X11 socket, so you can disable them

```
blacklist ${RUNUSER}/wayland-*
include disable-X11.inc
```

if you program does not need them.

Otherwise you need to run a nested server
- https://firejail.wordpress.com/documentation-2/x11-guide/
- https://github.com/netblue30/firejail/wiki/X11-Guide
- also do able for wayland but Ubu20.04 still sticks to X11 AFAIK.
